scss/jeet/_settings.scss

```diff
@@ -1,7 +1,5 @@
-/**
- * Grid settings.
- * All values are defaults and can therefore be easily overidden.
- */
+/// Grid settings.
+/// All values are defaults and can therefore be easily overidden.
 
```

Comment:
@jakecleary, I'm gonna need some help here. Every variable should be well documented, but I'm a little bit retarded to do that right now, at least in `$jeet-parent-first`, that I don't know what it does.